CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+## [1.14.0] - 2019-05-20
+### Added
+ * Expose `CallOptions` caller name for transparent proxying (#741)
+
+### Fixed
```

Comment:
can remove these, the relay path isn't part of the "public" API (muttley consumes `dev` directly) so no need to add these

Reply:
Sounds good, didn't realize this.